shared/dataflow/codeql/dataflow/VariableCapture.qll

```diff
@@ -0,0 +1,930 @@
+/**
+ * Provides a module for synthesizing data-flow nodes and related step relations
+ * for supporting flow through captured variables.
+ */
+
+private import codeql.util.Boolean
+private import codeql.util.Unit
+private import codeql.ssa.Ssa as Ssa
+
+signature module InputSig {
+  class Location {
```

Comment:
Can I? I can't really import that signature into this one, so that would mean splitting the input into two and parameterising this signature over the location-signature, and then I prefer keeping it as-is with just one input to the module.

Reply:
Yes, it would require using two module parameters.